src/components/Footer/Footer.js

```diff
@@ -340,7 +341,7 @@ const styles = StyleSheet.create({
   },
   text: {
     ...FONTS.body3,
-    color: COLORS.balance_gray
+    color: COLORS.gray400
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The text color has been changed from `COLORS.balance_gray` to `COLORS.gray400`. Ensure that `COLORS.gray400` is defined and provides the desired visual effect.
```diff
- color: COLORS.balance_gray
+ color: COLORS.gray400
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->